defaults/main/airflow-cfg.yml

```diff
@@ -21,25 +21,25 @@ airflow_database_schema:
 airflow_database_connect_args:
 airflow_parallelism: 32
 airflow_dag_concurrency: 16
-airflow_dags_are_paused_at_creation: true
+airflow_dags_are_paused_at_creation: True
```

Comment:
If this is meant to be a string rather than a boolean, I'd surround it with quotes to make it clear

Reply:
Just a boolean, but to be consistent we use True instead of true to look like in python